standalone/src/models/commons/price/price-draft/fields-config.ts

```diff
@@ -0,0 +1,46 @@
+import { type TModelFieldsConfig, fake } from '@/core';
+import { BaseMoneyDraftGraphql, MoneyDraftRest } from '@/models/commons';
+import { createRelatedDates } from '@/utils';
+import { TPriceDraftGraphql, TPriceDraftRest } from '../types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const commonFieldsConfig = {
+  fields: {
+    channel: null,
+    country: fake((f) => f.location.countryCode()),
+    custom: null,
+    customerGroup: null,
+    discounted: null,
+    key: null,
+    recurrencePolicy: null,
+    tiers: null,
+    validFrom: fake(getOlderDate),
+    validUntil: fake(getNewerDate),
+  },
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TPriceDraftRest> = {
+  fields: {
+    ...commonFieldsConfig.fields,
+    value: fake(() => MoneyDraftRest.random()),
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TPriceDraftGraphql> = {
+  fields: {
+    ...commonFieldsConfig.fields,
+    value: fake(() => BaseMoneyDraftGraphql.presets.withAllFields()),
```

Comment:
@commercetools/priceless-team-fe It would be great if you could let me know if I understood this one correctly or if it should be any different 🙏.

Reply:
Based on my previous comment, perhaps we could use one of the new proposed presets here and populate the value with a cent or high precision money.

For what I've seen in other models, usually we're populating with cents.

What do you think?